README.md

```diff
@@ -2,69 +2,110 @@
+| ----                | -----------                                                                                                                                                                                    |
+| CCP                 | Complete Curriculum Programme. A set of lesson materials and guidance that covers one subject for one Key Stage                                                                                |
+| Curriculum Designer | The person or persons responsible for creating a CCP                                                                                                                                           |
+| EAL                 | English as an additional language                                                                                                                                                              |
+| FSM                 | Free School Meals                                                                                                                                                                              |
+| LA                  | Local Authority e.g. Local Authority maintained schools                                                                                                                                        |
+| MAT                 | Multi-Academy Trusts                                                                                                                                                                           |
+| National curriculum | The national curriculum for England to be taught in all local-authority-maintained schools.                                                                                                    |
+| PPA                 | Planning, preparation and assessment                                                                                                                                                           |
+| Pedagogy            | Pedagogy, taken as an academic discipline, is the study of how knowledge and skills are imparted in an educational context, and it considers the interactions that take place during learning. |
+| SEN                 | Special Educational Needs or SEND, Special Educational Needs and Disabilities                                                                                                                  |
+| Unit                | A Unit of a CCP refers to the lessons that are delivered over the course of one half term. Normally 6 or 7 weeks                                                                               |
 
-The app should be available at https://govuk-rails-boilerplate.london.cloudapps.digital
+## Contributing
+
+Please ensure that any contributions meet with the [Department for Education's
+digital guidelines](https://dfe-digital.github.io/) and that all tests and lint checks
+pass.
+
+You can run them all with:
+
+```bash
+$ bundle exec rake
+```
```

Comment:
Include license

```suggestion
## 🎓 License

Unless stated otherwise, the codebase is released under the MIT License. This covers both the codebase and any sample code in the documentation. The documentation is © Crown copyright and available under the terms of the Open Government 3.0 licence.
```

Reply:
Added MIT for now, we won't need OGL [unless we start publishing documentation externally](https://gds-way.cloudapps.digital/manuals/licensing.html#copyright-notice)